app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -90,16 +92,22 @@ class PatientSearchResultItemView(
     }
   }
 
-  private fun renderPatientPhoneNumber(number: PhoneNumber) {
-    when (number) {
-      is PhoneNumber.Highlighted -> {
-        val highlightNumber = SpannableStringBuilder(number.patientNumber)
-        val highlightColor = context.resolveColor(colorRes = R.color.simple_light_blue_100)
-        highlightNumber.setSpan(BackgroundColorSpan(highlightColor), number.highlightStart, number.highlightEnd, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
-        phoneNumberTextView.text = highlightNumber
-      }
-      is PhoneNumber.Plain -> {
-        phoneNumberTextView.text = number.patientNumber
+  private fun renderPatientPhoneNumber(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel) {
+    phoneNumberContainer.visibleOrGone(patientSearchResult.phoneNumber.isNullOrBlank().not())
```

Comment:
Since this function is causing the code climate nested conditional issue, instead of separting them out to a separate function. we can do something like this (this solution is just for this scenario).

```kt
if (patientSearchResult.phoneNumber == null) {
  phoneNumberContainer.visibility = View.GONE
  return
}

phoneNumberContainer.visibility = View.VISIBLE
```